### ISSUE TITLE
The title of the "save" button in Preferences > Layouts is misleading

### ISSUE THREAD

**Issue by AlirezaHaghshenas** [gnome-terminator/terminator#858]
**Is your feature request related to a problem? Please describe.**
To a user, that might sound like it will save the changes we have made. But in reality, it will copy the current layout of the window (if we have multiple windows, it will copy from the open that triggered the preferences dialog).

**Describe the solution you'd like**
The button's title can be changed to something like "copy from window"

**Comment by RobQuistNL** [comment]
Yes. Ran into this multiple times. It threw away my old settings :(

**Comment by mattrose** [comment]
This is fixed in 2.1.5
